hw/ip/sysrst_ctrl/data/sysrst_ctrl_testplan.hjson

```diff
@@ -5,35 +5,220 @@
+              clear the interrupt.
+            '''
+      milestone: V2
+      tests: ["sysrst_ctrl_edge_detect"]
+    }
+
+    {
+      name: pin_output_keyboard_inversion_control
+      desc: '''
+            Verify the keyboard inversion feature by override logic.
+
+            * Select the output signals to override via PIN_OUT_CTL register. 
+            * Allow the output signals to override the value via PIN_ALLOWED_CTL register.
+            * Set the override value to the output signal via PIN_OUT_VALUE register.
+            '''
+      milestone: V2
+      tests: ["sysrst_ctrl_override_test"]
+    }
+
+    {
+      name: pin_input_value_accessibility
+      desc: '''
+            Verify the pin input value accessibilty.
+
+            * Trigger the key[0,1,2], ac_present_in, pwrb_in, ec_rst_in_l input pins with random values.
+            * Read the PIN_IN_VALUE register and check if the read value is same as input value.
+            '''
+      milestone: V2
+      tests: ["sysrst_ctrl_pin_access_test"]
+    }
+
+    {
+      name: ec_power_on_reset
+      desc: '''
+            Verify the EC and power on reset.
+
+            * Enable the override value by writing to PIN_ALLOWED_CTL.EC_RST_L_1 
+              and PIN_OUT_CTL.EC_RST_L register. 
+            * Make sure ec_rst_out_l is asserted even after opentitan reset is released.
+            * Set PIN_OUT_CTL.EC_RST_L to 0 to release the ec_rst_out_l reset.
+            '''
+      milestone: V2
+      tests: ["sysrst_ctrl_ec_pwr_on_rst"]
+    }
+
+    {
+      name: flash_write_protect_output
+      desc: '''
+            Verify the flash write protect.
+
+            * Make sure flash_wp_out_l signal is asserted low by reading PIN_OUT_CTL.flash_wp_l.
+            * Enable the override function by setting PIN_OUT_CTL.FLASH_WP_L to value 1. 
+            * Check if flash_wp_out_l is released. 
+            '''
+      milestone: V2
+      tests: ["sysrst_ctrl_flash_wr_prot_out"]
+    }
+
+    {
+      name: ultra_low_power_test
+      desc: '''
+            Verify the ultra low power feature. 
+
+            * Disable the bus clock to check detection logic works in sleep mode.
+            * Trigger the ac_present_i, lid_open_in, ec_rst_l_i input keys.         
+            * Configure ULP_AC_DEBOUNCE_CTL, ULP_LID_DEBOUNCE_CTL and 
+              ULP_PWRB_DEBOUNCE_CTL register to set the debounce timer value. 
+            * Turn on the bus clock to read the status register.                                                                                 
+            * Read the WKUP_STATUS register to check if the event has occured.                                                                                      
+            * Read the ULP_STATUS register and check if the ultra low power wakeup event is detected.
+            '''
+      milestone: V2
+      tests: ["sysrst_ctrl_ultra_low_pwr"]
+    }
+
+    {
+      name: stress_all_with_rand_reset
```

Comment:
It looks like CI complains about this test being a duplicate:
```
Error: Duplicate testpoint item found with name: stress_all_with_rand_reset
```

Afaik, this is a common test and does not have to be declared here (you can remove it).

Reply:
@weicaiyang is that right?

Reply:
you're right. This is already added from the imported testplan `stress_all_with_reset_testplan`. We should delete this entry